.github/workflows/workflow.yml

```diff
@@ -1,5 +1,8 @@
 name: Tests
 
+env:
+  ACTIONS_STEP_DEBUG: true
```

Comment:
I think you'll need to enable this via a Secret. See https://help.github.com/en/actions/configuring-and-managing-workflows/managing-a-workflow-run#enabling-debug-logging